src/FieldWarning/Assets/Ingame/UI/SelectionManager.cs

```diff
@@ -242,6 +248,9 @@ private void UnselectAll(List<PlatoonBehaviour> l, bool justPreviewing)
         private void SetSelected(List<PlatoonBehaviour> l, bool justPreviewing)
         {
             l.ForEach(x => x.SetSelected(true, justPreviewing));
+            // Randomly choose one platoon to play a selected voiceline
+            int r = Random.Range(0,l.Count);
+            l[r].PlaySelectionVoiceline();
```

Comment:
Not a fan of this kind of variable naming (regrettably exists all over the codebase). 

> Left[right].PlaySelectionVoiceline()? Ok, have to scroll up, now I know what you mean. Until I look at some other code. What was this a list of again?

It is not so bad in such a small method, but code grows over time and people don't always simplify it as they grow it (we can see this in action with your changes at lines 194-198 of the same file).

`_selection.Add(platoon)` is code that almost reads itself.